pydeseq2/DeseqStats.py

```diff
@@ -108,6 +120,19 @@ def __init__(
         ), "Please provide a fitted DeseqDataSet by first running the `deseq2` method."
 
         self.dds = dds
+
+        # Build contrast if None
+        if contrast is not None:
+            # TODO : tests on the contrast
+            self.contrast = contrast
```

Comment:
maybe check if the contrast is indeed given as required? ie three strings in a list and through an error otherwise

Reply:
Added tests in [39f4211](https://github.com/owkin/PyDESeq2/pull/32/commits/39f4211014399175ce08cc18e24141ffd5f08936) and [b0e35f5](https://github.com/owkin/PyDESeq2/pull/32/commits/b0e35f50c69f4c625a67896f4a9837972fc59fe9)